raft/log.go

```diff
@@ -937,12 +965,11 @@ func (l *Log) waitCommitted(index uint64) error {
 func (l *Log) waitUncommitted(index uint64) error {
 	for {
 		l.mu.Lock()
-		state, uncommittedIndex := l.state, l.index
+		uncommittedIndex := l.index
+		l.tracef("waitUncommitted: %s / %d", l.state, l.index)
 		l.mu.Unlock()
 
-		if state == Stopped {
```

Comment:
The `waitUncommitted()` is only for tests and it can be called before the log kicks into a follower state.